test/intergration/search/index_page.spec.js

```diff
@@ -0,0 +1,26 @@
+const assert = require('assert');
+const shunterTestHelper = require('shunter').testhelper();
+const fixtureHelper = require('../../helpers/fixture-helper');
+const paths = require('../../helpers/walk-helper');
+
+const b = require('js-beautify').html;
+
+describe('Search index page', function(){
+  before(function() {
+    shunterTestHelper.setup(paths);
+  });
+
+  after(shunterTestHelper.teardown);
+
+  it('should render content for search index page', function(done){
+    const jsonFixture = fixtureHelper.getJSONFixture('index_page');
```

Comment:
could we have something like the following, in addition to the individual methods to help clean this up? It may make the tests more readable when your HTML and JSON are named the same:

```javascript
const fixtures = fixtureHelper.getFixtures('index_page');

// usage
  shunterTestHelper.render('layout', fixtures.json, function(error, dom, output){
    assert.strictEqual(b(fixtures.html), b(output));
     done();
   });
```